radarr/templates/sqp/sqp-2.yml

```diff
@@ -15,111 +15,25 @@ radarr:
     base_url: Put your Radarr URL here
     api_key: Put your API key here
 
-# Quality Definition
-# This section defines the Quality Definitions that will be used in your Radarr instance
-# Only one set of Quality Definitions can be used per instance
-# Documentation: https://recyclarr.dev/wiki/yaml/config-reference/quality-definition/
-    quality_definition:
-      type: sqp-uhd
-
-# Quality Profiles
-# This section creates and/or updates Quality Profile(s) in your Radarr instance
-# Documentation: https://recyclarr.dev/wiki/yaml/config-reference/quality-profiles/
-    quality_profiles:
-      - name: SQP-2
-        reset_unmatched_scores:
-          enabled: true
-        upgrade:
-          allowed: true
-          until_quality: WEB|Remux|Bluray|2160p
-          until_score: 10000
-        min_format_score: 550
-        score_set: sqp-2
-        quality_sort: top
-        qualities:
-          - name: WEB|Remux|Bluray|2160p
-            qualities:
-              - WEBDL-2160p
-              - WEBRip-2160p
-              - Bluray-2160p
-              - Remux-2160p
-              - WEBDL-1080p
-              - WEBRip-1080p
-          - name: Remux-1080p
+    include:
+      - template: sqp-2
 
 # Custom Formats
 # This section creates and/or updates Custom Formats in your Radarr instance,
 # and applies them and their scores to Quality Profiles
 # Documentation: https://recyclarr.dev/wiki/yaml/config-reference/custom-formats/
+
+# You must comment out lines 27, 28, 49 and 50 if you disable all the custom formats below
```

Comment:
This is actually a really interesting use case. I think I will update Recyclarr to allow empty `trash_ids` lists, in order to make the editing experience of the templates a little better.

Reply:
That would definitely clean up this area of the config, thanks!